src/features/common/Layout/Header/index.tsx

```diff
@@ -96,5 +97,7 @@ export default function Header() {
         <meta name="theme-color" content={styles.primaryColor} />
       ) : null}
     </Head>
+    <noscript>This browser does not support our website</noscript>
```

Comment:
Why didn't you use the wording from the issue description at https://github.com/Plant-for-the-Planet-org/planet-webapp/issues/975#issue-855395884?